cve_bin_tool/strings.py

```diff
@@ -33,7 +33,7 @@ async def aio_parse(self):
                     elif tmp:
                         self.output.append("".join(tmp))
                         tmp = []
-        return self.output
+        return "\n".join(self.output)
```

Comment:
This is not the optimal solution.
I tried 
```
def __init__(self, filename=""):
        self.filename = filename
        self.output = ""                   <<

    async def aio_parse(self):
        async with FileIO(self.filename, "rb") as f:
            tmp = []
            async for line in f:
                for char in line:
                    # remove all unprintable characters
                    if char in Strings.PRINTABLE:
                        tmp.append(chr(char))
                    elif tmp:
                        self.output += "".join(tmp)        <<
                        tmp = []
        return self.output
```
and 
```
def __init__(self, filename=""):
        self.filename = filename
        self.output = ""                   <<

    async def aio_parse(self):
        async with FileIO(self.filename, "rb") as f:
            tmp = []
            async for line in f:
                for char in line:
                    # remove all unprintable characters
                    if char in Strings.PRINTABLE:
                        tmp.append(chr(char))
                    elif tmp:
                        self.output += "\n".join(tmp)        <<
                        tmp = []
        return self.output
```
but the 1st code doesn't detect multiline regexes and 2nd doesn't detect anything.